pysatMissions/instruments/missions_ephem.py

```diff
@@ -213,38 +223,38 @@ def load(fnames, tag=None, inst_id=None, obs_long=0., obs_lat=0., obs_alt=0.,
                          index=index)
     data.index.name = 'Epoch'
 
-    return data, meta.copy()
+    meta = pysat.Meta()
+    meta['Epoch'] = {
+        meta.labels.units: 'Milliseconds since 1970-1-1',
+        meta.labels.notes: 'UTC time at middle of geophysical measurement.',
+        meta.labels.desc: 'UTC seconds',
+        meta.labels.name: 'Time index in milliseconds'}
+    meta['glong'] = {meta.labels.units: 'degrees',
```

Comment:
Parameters should have meta.labels.min_val, meta.labels.max_val, meta.labels.fill_val